src/RefractAPI.cc

```diff
@@ -6,7 +6,6 @@
 //  Copyright (c) 2015 Apiary Inc. All rights reserved.
 //
 
-#include <iterator>
```

Comment:
Ah that's a good point. It's no longer been a problem in Objective-C with the introduction of clang modules. I completely forgot it's not standard/default across all compilers.

:+1: